### PR TITLE
Feature/feedback

### PR DESCRIPTION
**Pull request by jaliquiel**
**What this PR does**:
adds xml files for feedback screen: 
https://app.zeplin.io/project/5ec4729feee8e847b8d9f1d1/screen/5f24ef9ad3d1fd1f74ef51f1

**Which issue(s) this PR fixes**:
#121 

Fixes #

**Special notes for reviewers**:

**Additional comments**:

### CODE REVIEW

**Comment by jaliquiel** [comment]
Here are screenshots of the feedback xml screen:
Top
image

Scrolled down
image

**Review by mayorjay — APPROVED** [review]

File: profile/src/main/res/layout/feedback_fragment.xml [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+
+        </androidx.constraintlayout.widget.ConstraintLayout>
+
+
+    </ScrollView>
+
+    <View
+        android:id="@+id/bottom"
+        android:layout_width="wrap_content"
+        android:layout_height="1dp"
```
Comment (carl05) [inline profile/src/main/res/layout/feedback_fragment.xml]: use dimens.xml atribute

File: profile/src/main/res/layout/feedback_needs_rating.xml [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <RadioGroup
+        android:id="@+id/rating_radio_group"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="16dp"
+        android:layout_marginTop="16dp"
+        android:layout_marginEnd="16dp"
```
Comment (carl05) [inline profile/src/main/res/layout/feedback_needs_rating.xml]: use dimens.xml atribute

File: profile/src/main/res/layout/feedback_needs_rating.xml [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+        android:orientation="horizontal"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.498"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/rating_text">
+
+        <RadioButton
+            android:id="@+id/radio0"
+            style="@style/RatingRadioButton"
+            android:layout_width="0dp"
```
Comment (carl05) [inline profile/src/main/res/layout/feedback_needs_rating.xml]: use dimens.xml atribute

File: profile/src/main/res/layout/feedback_needs_rating.xml [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+            android:layout_height="wrap_content"
+            android:layout_weight="1"
+            android:text="@string/rating2"
+            android:theme="@style/RatingRadioButtonColor" />
+
+        <com.google.android.material.radiobutton.MaterialRadioButton
+            android:id="@+id/rating_3"
+            style="@style/RatingRadioButton"
+            android:layout_width="0dp"
```
Comment (carl05) [inline profile/src/main/res/layout/feedback_needs_rating.xml]: use dimens.xml atribute

File: profile/src/main/res/layout/feedback_question1.xml [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+    android:layout_width="fill_parent"
+    android:layout_height="wrap_content"
+    android:animateLayoutChanges="true"
+    card_view:cardBackgroundColor="@color/colorWhite"
+    card_view:cardCornerRadius="0dp"
+    card_view:cardElevation="0dp">
+
+    <FrameLayout
+        android:id="@+id/verticalAccent1"
+        android:layout_width="3dp"
```
Comment (carl05) [inline profile/src/main/res/layout/feedback_question1.xml]: use dimens.xml atribute

File: profile/src/main/res/layout/feedback_question1.xml [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+        android:layout_height="match_parent"
+        android:background="@color/fightPandemicsNeonBlue"
+        android:visibility="gone"
```
Comment (carl05) [inline profile/src/main/res/layout/feedback_question1.xml:17]: why different visibilities?

File: profile/src/main/res/layout/feedback_question2.xml [hunk 7]
```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<com.google.android.material.card.MaterialCardView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:card_view="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="fill_parent"
+    android:layout_height="wrap_content"
+    android:animateLayoutChanges="true"
+    card_view:cardBackgroundColor="@color/colorWhite"
+    card_view:cardCornerRadius="0dp"
+    card_view:cardElevation="0dp">
```
Comment (carl05) [inline profile/src/main/res/layout/feedback_question2.xml]: use dimens.xml atribute

File: profile/src/main/res/layout/feedback_question3.xml [hunk 8]
```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<com.google.android.material.card.MaterialCardView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:card_view="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="fill_parent"
+    android:layout_height="wrap_content"
+    android:animateLayoutChanges="true"
+    card_view:cardBackgroundColor="@color/colorWhite"
+    card_view:cardCornerRadius="0dp"
+    card_view:cardElevation="0dp">
```
Comment (carl05) [inline profile/src/main/res/layout/feedback_question3.xml]: use dimens.xml atribute

**Review by carl05 — CHANGES_REQUESTED** [review]

File: profile/src/main/res/layout/feedback_question1.xml (hunk 6 above)
Comment (jaliquiel) [reply profile/src/main/res/layout/feedback_question1.xml:17]: It's just so it appears in the XML editor. The view model makes this appear and disappear.

Comment (jaliquiel) [reply profile/src/main/res/layout/feedback_question1.xml:17]: Should I remove it or is it fine?

Comment (carl05) [reply profile/src/main/res/layout/feedback_question1.xml:17]: Its ok

**Review by carl05 — APPROVED** [review]

**Pull request merged**